integration/spark/src/main/scala/org/apache/spark/sql/execution/command/schema/CarbonAlterTableDropColumnCommand.scala

```diff
@@ -147,19 +143,30 @@ private[sql] case class CarbonAlterTableDropColumnCommand(
         metastore.getThriftTableInfo(carbonTable)
       // maintain the deleted columns for schema evolution history
```

Comment:
please edit comment saying `deletedColumnSchema ` contains parent column schema and child also in case of complex and `deletedTableColumns ` contains only parent columns in case of complex to add in schemaEvolution entry

Reply:
updated